test #18 fails building 0.10.1

possibly to do with the fact I'm doing it through debian

```
18/18 Test #18: mime-roundtrip ......................................***Failed    0.01 sec
Running 4 test cases...
Can't open file: TestMessages/00000001
/usr/local/src/cpp-netlib-0.10.1/libs/mime/test/mime-roundtrip.cpp(71): fatal error in "boost::bind ( test_roundtrip, "TestMessages/00000001" )": exception thrown by mp = to_mime ( fileName )
Can't open file: TestMessages/00000019
/usr/local/src/cpp-netlib-0.10.1/libs/mime/test/mime-roundtrip.cpp(71): fatal error in "boost::bind ( test_roundtrip, "TestMessages/00000019" )": exception thrown by mp = to_mime ( fileName )
Can't open file: TestMessages/00000431
/usr/local/src/cpp-netlib-0.10.1/libs/mime/test/mime-roundtrip.cpp(71): fatal error in "boost::bind ( test_roundtrip, "TestMessages/00000431" )": exception thrown by mp = to_mime ( fileName )
Can't open file: TestMessages/00000975
/usr/local/src/cpp-netlib-0.10.1/libs/mime/test/mime-roundtrip.cpp(71): fatal error in "boost::bind ( test_roundtrip, "TestMessages/00000975" )": exception thrown by mp = to_mime ( fileName )

*** 4 failures detected in test suite "Master Test Suite"


94% tests passed, 1 tests failed out of 18

Total Test time (real) =  20.19 sec

The following tests FAILED:
     18 - mime-roundtrip (Failed)
Errors while running CTest
make[1]: *** [test] Error 8
make[1]: Leaving directory `/usr/local/src/cpp-netlib-0.10.1/obj-x86_64-linux-gnu'
dh_auto_test: make -j1 test ARGS+=-j1 returned exit code 2
make: *** [build] Error 2
dpkg-buildpackage: error: debian/rules build gave error exit status 2
debuild: fatal error at line 1357:
dpkg-buildpackage -rfakeroot -D -us -uc failed
29
```

--- a/libs/mime/test/CMakeLists.txt
+++ b/libs/mime/test/CMakeLists.txt
@@ -1,5 +1,6 @@
 include_directories(${CPP-NETLIB_SOURCE_DIR})
 find_package ( Boost 1.41.0 COMPONENTS unit_test_framework )
+file ( COPY TestMessages DESTINATION ${CMAKE_CURRENT_BINARY_DIR} )
 
 option (BOOST_DYN_LINK
   "Build the project using dynamic linking for the boost test libs"

